upgrade-tidb-using-tiup.md

```diff
@@ -136,7 +140,7 @@ Now, the offline mirror has been upgraded successfully. If an error occurs durin
 >
 > Before you upgrade the cluster to v6.5.0, make sure that the parameters you have modified in v4.0 are compatible in v6.5.0. For details, see [TiKV Configuration File](/tikv-configuration-file.md).
 
-### Step 3: Check the health status of the current cluster
+### Step 5: Check the health status of the current cluster
```

Comment:
```suggestion
### Step 4: Check the health status of the current cluster
```

Reply:
Change [Line 158](https://github.com/pingcap/docs/pull/12408/files#diff-e210c3a991b4061481b908eed6924a0b5c8acfa7528d7c731b3308321f811a97R158) to Step 5.

Reply:
Done. Thanks